### ISSUE TITLE
Looking for help?

### ISSUE THREAD

**Issue by jsmestad** [elixir-buildpack/cloud-native-buildpack#1]
I'd love to make this project a reality. Is there anywhere I can help?

**Comment by KazW** [comment]
Absolutely! I never really got an environment working where I could actually deploy apps, getting Kubernetes setup the way I wanted proved to be more of a challenge than I wanted to tackle in my spare time. The code in this repo should work in theory, or at least as a starting point, it'll download OTP and Elixir, and compile an app.
